### ISSUE TITLE
Unplugin-vue-single-define fails with two script tags

### ISSUE THREAD

**Issue by matthew-dean** [sxzz/unplugin-vue-macros#322]
### Describe the bug

I got this error:
```
[plugin:unplugin-vue-single-define] Unexpected token, expected "from" (2:12)
.../ui/src/components/ds/IconButton.vue:2:12
1  |  <script setup lang="ts">
```
... as soon as I added:
```vue
<script>
export default {
  inheritAttrs: false
}
</script>
```
Is the use of `defineOptions` an absolute requirement?

### Reproduction

just put in <script setup> and <script>

### System Info

```Shell
System:
    OS: macOS 12.3
    CPU: (10) arm64 Apple M1 Pro
    Memory: 77.88 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.19.0 - ~/Library/Caches/fnm_multishells/8259_1679444896232/bin/node
    npm: 8.19.3 - ~/Library/Caches/fnm_multishells/8259_1679444896232/bin/npm
  Browsers:
    Chrome: 111.0.5563.110
    Safari: 15.4
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/sxzz/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/sxzz/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sxzz** [comment]
I didn't get your point. Please provide more detail and code.

Note: Using two script blocks is not recommended while using Vue Macros. Please try to use `<script setup>` only instead.

**Comment by sxzz** [comment]
We temporarily close this due to the lack of enough information. Please provide a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) to reopen the issue. Thanks.

https://antfu.me/posts/why-reproductions-are-required

**Comment by matthew-dean** [comment]
> Note: Using two script blocks is not recommended while using Vue Macros. 

Why? This is a perfectly valid scenario that Vue SFCs support.

**Comment by sxzz** [comment]
Yes, it's just a suggestion which means it's not first-class support. Please provide a reproduction and I can figure out the problem.

**Comment by ipcjs** [comment]
It looks like this is an error message prompt issue.

When the language of two `<script>` does not match, without the `reactivity-transform` plugin enabled, `vite` prompts:
`[plugin:vite:vue] [@vue/compiler-sfc] <script> and <script setup> must have the same language type.`

If the `reactivity-transform` plugin is enabled, `vite` prompts for:
`[plugin:unplugin-reactivity-transform] Unexpected token, expected "," (3:8)`

### minimal reproducible code
```
<script>
export default {}
</script>
<script setup lang="ts">
defineProps<{
    size?: string | number
    minSize?: string | number
}>()
</script>

<template>
    <div> {{ size }} {{ minSize }} </div>
</template>
```

**Comment by sxzz** [comment]
Fixed in unplugin-vue-macros@2.1.
